spec/lib/lighthouse/facilities/client_spec.rb

```diff
@@ -56,22 +57,24 @@
         'tuesday' => '730AM-430PM',
         'wednesday' => '730AM-430PM'
       },
-      services: { 'health' => %w[Audiology Cardiology Dermatology Ophthalmology
-                                 PrimaryCare SpecialtyCare],
-                  'last_updated' => '2020-09-14', 'other' => [] },
+      services: { 'health' => %w[Audiology Cardiology Dermatology Gastroenterology
+                                 MentalHealthCare Ophthalmology PrimaryCare SpecialtyCare],
+                  'last_updated' => '2021-01-25', 'other' => [] },
       feedback: {
         'effective_date' => nil,
         'health' => {}
       },
       access: {
-        'effective_date' => '2020-09-14',
+        'effective_date' => '2021-01-25',
```

Comment:
Does the effective_date attribute (and others that are still snake) need to be camel cased here?

Reply:
Good catch, this is a bug in fast_jsonapi and a larger argument. let me see what I can do. [Deep transform keys for complex attributes](https://github.com/jsonapi-serializer/jsonapi-serializer/issues/62)

Reply:
Awesome 👍

Reply:
I see the changes for deep_stringify_keys.deep_transform_keys in the serializer. Is this what you were referring to when you said that anything uploaded still need to be snake cased?